src/goto-instrument/contracts/contracts.cpp

```diff
@@ -34,6 +34,12 @@ Date: February 2016
 #include <util/pointer_offset_size.h>
 #include <util/replace_symbol.h>
 
+// Create a lexicographic less-than relation between two tuples of variables.
+// This is used in the implementation of multidimensional decreases clauses.
+exprt create_lexicographic_less_than(
+  const std::vector<symbol_exprt> &lhs,
+  const std::vector<symbol_exprt> &rhs);
```

Comment:
Function declarations should go to header files.

Reply:
I don't want the function `create_lexicographic_less_than` to be automatically included whenever the header file is included. As of now, this function is only used in the implementation of multidimensional decreases clauses. So I am basically the only client of this function. Any suggestion?

Reply:
@LongPham7 take a look at [static functions](https://www.geeksforgeeks.org/what-are-static-functions-in-c/).

Reply:
OK. I will make the function static and put the function declaration in the header. Thanks for the suggestion!